src/models/user.js

```diff
@@ -0,0 +1,70 @@
+import mongoose from 'mongoose';
+const { Schema, model } = mongoose;
+
+const userSchema = new Schema({
+  mail: {
+    type: String,
+    required: true,
+    default: null,
+    trim: true,
+  },
+  name: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  displayPicture: {
+    type: String,
+    required: false,
+    // We can have a default image
+  },
+  firebaseID: {
+    type: String,
+    required: true,
+  },
+  accessLevel: {
+    type: String,
```

Comment:
make it into an array of objects

Reply:
so as in, one user can have multiple accesslevels?